packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -145,9 +145,11 @@ export class RuxModal {
                 <Host>
                     <div part="wrapper" class="rux-modal__wrapper">
                         <dialog class="rux-modal__dialog" role="dialog">
-                            <header class="rux-modal__titlebar">
-                                <div>{modalTitle}</div>
-                            </header>
+                            {modalTitle && (
+                                <header class="rux-modal__titlebar">
+                                    <div>{modalTitle}</div>
+                                </header>
+                            )}
                             <div class="rux-modal__content">
                                 <div class="rux-modal__message">
                                     {modalMessage}
```

Comment:
Specifically for the main content or `message` portion of the modal, should we have a slot and/or a prop? That would help turn it into a container for more than just text.
```suggestion
                                    <slot>{modalMessage}</slot>
```